Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -132,10 +132,10 @@ private string ConvertJToken(JToken jToken)
                         _ => value?.ToString()?.Replace(',', '.'), // Replace decimal commas.
                     };
                 case JArray jArray:
-                    return $"new[] {{ {string.Join(", ", jArray.Select(ConvertJToken))} }}";
+                    return $"new[] \n\t\t\t{{ {string.Join(", ", jArray.Select(ConvertJToken))} \n\t\t\t}}";

```

Comment:
Why is the additional whitespace needed?

Reply:
Strictly to format the output. Without it, the output looked very wacky. Happy to provide a screenshot if you like.

Reply:
Without the whitespace it looks like this:

![image](https://user-images.githubusercontent.com/237369/110207386-3addd780-7e51-11eb-9e10-de4ae02502e1.png)

Reply:
Ah, I see now. However, the issue I see is that we don't just generate this for `ListValueOption` as far as I remember, other settings can have arrays too. And if it's just three strings or so you want that in a single line.

Reply:
Ok I will see if I can only apply the formatting if it's a `ListValueOption`

Reply:
Updated formatting to online indent for a `JObject`.

![image](https://user-images.githubusercontent.com/237369/114092040-6eb08080-9887-11eb-9864-74f6cc2f7afb.png)